tests/functional/services/auth/test_auth_client_flow.py

```diff
@@ -55,23 +56,24 @@ class CustomAuthClient(globus_sdk.ConfidentialAppAuthClient):
     ["baz-id", "quux-id"],
     ["baz-id", uuid.UUID(int=5)],
 )
+mfa_options = (True, False)
 prompt_options = ("login",)
 # Seed an all-`None` option test, then use a loop to fill in the rest.
 # There must be 4 parameters so parametrized tuple unpacking works in the test.
```

Comment:
```suggestion
# There must be 5 parameters so parametrized tuple unpacking works in the test.
```

I feel like there's a rephrase such that the number isn't repeated here... Maybe just this?

```suggestion
# The number of parameters here must match the test parameters.
```